front/app/modules/commercial/insights/services/insightsCategoriesSuggestionsTasks.ts

```diff
@@ -84,10 +84,10 @@ export async function insightsTriggerCategoriesSuggestionsTasks(
       // Refetch inputs when there are no pending tasks
       finalize(() => {
         streams.fetchAllWith({
-          partialApiEndpoint: [
-            `insights/views/${insightsViewId}/inputs`,
-            `insights/views/${insightsViewId}/categories`,
+          apiEndpoint: [
+            `${API_PATH}/insights/views/${insightsViewId}/categories`,
```

Comment:
Made this a strictly defined endpoint to avoid refetching individual categories that match the same path